include/freetds/thread.h

```diff
@@ -353,6 +353,14 @@ static inline int tds_mutex_trylock(tds_mutex *mtx)
 	assert(mtx);
 	ret = tds_raw_mutex_trylock(&mtx->mtx);
 	if (!ret) {
+#ifdef _WIN32
+		/* disallow recursive locking! */
```

Comment:
This part of code affects only debug build, is this expected?
In this case I would just assert so to catch the issues during testing.

Reply:
Now on Windows mutexes also are not recursive so this patch is not necessary anymore.

Reply:
OK, thanks!